themes/digital.gov/src/scss/new/_gh-edit-page.scss

```
@@ -0,0 +1,145 @@
+@use "uswds-core" as *;
+
+/*
+  gh-edit-page applies yellow highlight box for components that use the attribute [gh-edit-page]
+  for linking to github repo link
+*/
+
+.gh-edit-page {
+  background-color: color("yellow-5");
+  position: relative;
+
+  &__button {
+    @include u-font("sans", "2xs");
+    background-color: color("gold-20v");
+    border-radius: 99rem;
+    box-shadow: 0 1rem 2rem 0 rgba(0, 0, 0, 0.1);
+    color: color("gray-1");
+    height: 1rem;
+    margin-top: 0;
+    padding: 0;
+    position: absolute;
+    right: -10px;
+    text-align: center;
+    text-decoration: none;
+    top: -10px;
+    width: 1rem;
+    z-index: 100;
+  }
+
+  &__button-text {
+    display: none;
+
+    &:hover {
+      background-color: color("gold-20v");
+      text-decoration: underline;
+    }
+  }
+
+  &:hover {
+    outline: units(0.5) color("yellow-20v") solid;
+    z-index: 100;
+
+    .gh-edit-page__button {
+      color: white;
+      display: block;
+      height: 40px;
+      right: -20px;
+      top: -20px;
+      width: 40px;
+    }
+
+    .gh-edit-page__button-text {
+      border: none;
+      border-radius: 99rem;
+      display: block;
+      padding: 9px 6px;
+      position: relative;
+      text-indent: 0;
+    }
+  }
+}
+
+/*
+  .gh-edit-tools styles the bottom right corner edit tools to enable github edit button and report an issue button
+*/
+
+.gh-edit-tools {
+  display: none;
+  position: relative;
+
+  @include at-media("tablet") {
+    display: block;
+  }
+
+  &__button {
+    @include u-font("sans", "sm");
+    background-color: color("blue-50v");
+    border: 0;
+    border-radius: 99rem;
+
+    bottom: 20px;
+    box-shadow: 0 1rem 2rem 0 rgba(0, 0, 0, 0.1);
+    color: white;
+    display: block;
+    height: 4.5rem;
+    padding: 20px;
+    position: fixed;
+    right: 20px;
+    text-align: center;
+    text-decoration: none;
+    width: 4.5rem;
+    z-index: 99999;
+
+    &:hover {
+      background-color: color("blue-60v");
+    }
+
+    &:focus {
+      background-color: color("blue-60v");
+    }
+  }
+
+  &__issue-link {
+    display: none;
+  }
+}
+
+.gh-edit-tools--active {
+  .gh-edit-tools__button {
+    @include u-font("sans", "lg");
+    background-color: white;
+    border: units(1) color("blue-50v") solid;
+    color: color("blue-50v");
+    padding: 12px;
+
+    &:hover {
+      background-color: color("gray-3");
+    }
+
+    &:focus {
+      background-color: color("gray-3");
+    }
```

You could just combine these.

```suggestion
    &:hover,
    &:focus {
      background-color: color("gray-3");
    }
```